board/netconf/rootfs/libexec/infix/ospf-status

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+tempfile=`mktemp`
+vtysh -c "show ip ospf interface json" | jq '[.interfaces as $interfaces | $interfaces | to_entries | group_by(.value.area)[] | {(.[0].value.area): [.[] | {name: .key} + .value] }] | reduce .[] as $item ({}; . + $item)' > $tempfile
```

Comment:
Maybe we should start writing some comments for jq expressions like this one?  "Purpose of this script is to ... "

Reply:
Sounds good, that script is unreadable.